python/src/naive_api_client.py

```diff
@@ -45,13 +45,17 @@ class BridgeTokenRequest(TypedDict):
             product_type: str
             client_name: str
             tracking_info: str
+            account: TypedDict
 
         request_data: BridgeTokenRequest = {
             'product_type': self.PRODUCT_TYPE,
             'client_name': 'Citadel Quickstart',
             'tracking_info': '1337'
         }
 
+        if self.PRODUCT_TYPE == 'fas':
+            request_data['account'] = account            
```

Comment:
can we define account here, as other parameters? (instead of function parameter)